src/textbooks/Textbooks.scss

```
@@ -1,3 +1,8 @@
+@import "./scss/variables";
 @import "./empty-placeholder/EmptyPlaceholder";
 @import "./textbook-card/TextbookCard";
 @import "./textbook-form/TextbookForm";
+
+.alert-toast {
+  z-index: $popover !important;
```

[important]: Try to use Paragon SCSS variable

```suggestion
  z-index: $zindex-tooltip !important;
```

---

@PKulkoRaccoonGang We have the same variable for certificates page, should we change there also?

---

i remember that you suggested exactly this naming so im curious

---

Fixed